src/main/java/io/projectreactor/ModuleUtils.java

```diff
@@ -102,6 +103,54 @@ public static void loadModuleVersionsFromArtifactoryVersionsSearch(String json,
 		versions.forEach(v -> tryAddVersion(module, v));
 	}
 
+	public static void fetchVersionsFromSonotype(Map<String, Module> modules, String... moduleNames) {
```

Comment:
I guess there's a typo with the method name. Should it be `sonatype` instead?
```suggestion
	public static void fetchVersionsFromSonatype(Map<String, Module> modules, String... module names) {
```